.github/actions/cache-ui-dependencies/action.yaml

```diff
@@ -24,4 +24,3 @@ runs:
         restore-keys: |
           npm-v2-${{ hashFiles(inputs.lockFile) }}-${{ github.job }}
           npm-v2-${{ hashFiles(inputs.lockFile) }}-
-          npm-v2-
```

Comment:
Could a `${{ github.branch (need to verify that field exists) }}` be added to the restore-key for this final match?

Reply:
@gavin-stackrox I think that using lock files for cache matching is the only right way. Everything else could lead to wrong cache hits. And maybe also the NodeJS version if it's not already part of `ui/monorepo.lock`.

If we would use a branch name for matching cache. We could hit cache in the same branch after lock files are changed.

Reply:
Ok.